frontend/src/components/maps/hooks/useActiveFeatureLayer.tsx

```diff
@@ -102,6 +115,12 @@ const useActiveFeatureLayer = ({
         ...feature.properties,
         IS_SELECTED: isSelecting,
         CLICK_LAT_LNG: latLng,
+        REGION_NUMBER: isNumber(region.REGION_NUMBER) ? region.REGION_NUMBER : RegionCodes.Unknown,
+        REGION_NAME: region.REGION_NAME ?? 'Cannot determine',
+        DISTRICT_NUMBER: isNumber(district.DISTRICT_NUMBER)
+          ? district.DISTRICT_NUMBER
+          : DistrictCodes.Unknown,
+        DISTRICT_NAME: district.DISTRICT_NAME ?? 'Cannot determine',
```

Comment:
Acceptance criteria call for:
* If no match is found map it to Type Unknown - "Can not determine."